react/components/LogoutCustomerSession.tsx

```diff
@@ -49,6 +53,24 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm();
+
+  const id = orderForm.id;
+
+  const [customerClass, setCustomerClass] = useState("");
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      var requestOptions = {
+        method: 'GET'
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
+  
+    getCustomerClass().then(response => response.json()).then(data => {
+      setCustomerClass(data.clientProfileData.customerClass);
```

Comment:
What if `clientProfileData` or `customerClass` don't exist?

Reply:
Updated the code to take care of this issue

Reply:
@lariciamota Updated the code to take care of this issue